admin_guide/native_routing.adoc

```diff
@@ -0,0 +1,68 @@
+= Native container routing using network infrastructure
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+
+== Introduction
+This document describes how one could setup container networking using existing switches/routers and using
+the kernel networking stack in Linux. The setup requires that the network administrator or some script
+modifies the router[s] as new nodes are added to the cluster. The document will describe the steps to
+use a Linux server as a simple router. The steps could then be adapted to any particular router.
+
+
+== Network Layout
+The diagram below shows the setup used in the document. It has one Linux node with two network interface
+cards serving as a router, two switches and three nodes connected to these switches. 
+
+image::network_diagram.png["Network Diagram"]
```

Comment:
@mrunalp Sorry, just noticed: the PNG file itself should go in an `admin_guide/images/` folder for it to show up in the build. You can still reference the file here as you are.